avro-java8/pom.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion> 
+  <parent>
+    <groupId>com.fasterxml.jackson.dataformat</groupId>
+    <artifactId>jackson-dataformats-binary</artifactId>
+    <version>2.10.0.pr2-SNAPSHOT</version>
+  </parent>
+  <artifactId>jackson-dataformat-avro-java8</artifactId>
+  <name>Jackson dataformat: Avro Java 8</name>
+  <packaging>bundle</packaging>
+  <description>Support for reading and writing AVRO-encoded data via Jackson
+abstractions.
+  </description>
+  <url>http://github.com/FasterXML/jackson-dataformats-binary</url>    
+
+  <properties>
+    <!-- Generate PackageVersion.java into this directory. -->
+    <packageVersion.dir>com/fasterxml/jackson/dataformat/avro/java8</packageVersion.dir>
+    <packageVersion.package>${project.groupId}.avro.java8</packageVersion.package>
+  </properties>
+
+  <dependencies>
+    <!--  Hmmh. Need annotations for introspection... -->
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-annotations</artifactId>
+    </dependency>
+    <!--  and databind for Avro Schema generation... -->
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.dataformat</groupId>
+      <artifactId>jackson-dataformat-avro</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+
+    <!-- and for testing we need logback -->
+    <dependency>
+      <groupId>ch.qos.logback</groupId>
+      <artifactId>logback-classic</artifactId>
+      <version>1.1.3</version>
+      <scope>test</scope>
+    </dependency>
+    <!-- A bit of help to reduce boiler-plate in dummy test classes -->
+    <dependency>
```

Comment:
Quick note: no, unfortunately I will not allow Lombok as a dependency (some usage did sneak in, and I have been busy removing it). Originally reason was that Lombok use prevented clean build out of cloned repo (some component had to be locally installed).